yarn.lock

```diff
@@ -4,42 +4,42 @@
 
 "@babel/code-frame@7.12.11":
   version "7.12.11"
-  resolved "https://registry.yarnpkg.com/@babel/code-frame/-/code-frame-7.12.11.tgz#f4ad435aa263db935b8f10f2c552d23fb716a63f"
+  resolved "https://registry.npmjs.org/@babel/code-frame/-/code-frame-7.12.11.tgz"
```

Comment:
Why are these changing form `yarnpkg.com` to `npmjs.org` ? Are you using `yarn` or `npm`?

And do you know why the integrity hashes are getting dropped in the resolved URLs?

Would prefer to keep them.

Reply:
I think I used npm to install prettierjs or something.

Reply:
How am I supposed to fix the integrity hashes and the npm thing?

Reply:
It says `DO NOT EDIT THIS FILE DIRECTLY.`

Reply:
There are multiple ways you can do this;

One way is to undo some of your commits or the commits to this file via `git reset --soft HEAD^` (which will undo all of the files and put them in the `staged` stage)

Then, in the future, do not use `npm` anymore; going forward, just use `yarn add` instead of `npm install`

The broader JS community has moved on to using `yarn` as the de facto standard, rather than `npm`